lnwallet/chanfunding/psbt_assembler.go

```diff
@@ -319,32 +339,22 @@ func (i *PsbtIntent) CompileFundingTx() (*wire.MsgTx, error) {
 			i.State, PsbtFinalized)
 	}
 
-	// Make sure the PSBT can be finalized and extracted.
```

Comment:
are these checks moved elsewhere, or removed intentionally ?

Reply:
Before, we used to store the PSBT in the intent until `CompileFundingTx()` was called from the funding manager. So we always had to extract the final transaction there (even though that was already checked in `Finalize()`). But since we now store the finalized and extracted raw transaction in the intent, this extra check and extraction is not necessary anymore.